src/monitors.ts

```diff
@@ -84,7 +85,7 @@ export function buildMonitorParams(
       monitorParams.message = recommendedMonitor.message;
     }
     if (!monitorParams.name) {
-      monitorParams.name = interpolateTemplateVariables(recommendedMonitor.templateVariables, recommendedMonitor.name);
+      monitorParams.name = interpolateTemplateVariables(recommendedMonitor, recommendedMonitor.name);
```

Comment:
**Nit:** It seems odd that we're both passing the entire recommended monitor as well as the name in a separate param. This function could simply take the recommended monitor alone and read the name from it.

**Another Nit:** With the changes done, a name such as `getInterpolatedMonitorName` might be easier to understand than `interpolateTemplateVariables`